languages/js/package.json

```diff
@@ -39,6 +39,8 @@
   },
   "devDependencies": {
     "@types/jest": "^26.0.9",
+    "@types/lodash": "^4.14.175",
```

Comment:
why do we need this?

Reply:
can't import from an external package without it.

Reply:
like we need the base `@types/lodash` in order to import `@types/lodash.isequal`?

Reply:
ah I see:
```
+  dependencies:
+    "@types/lodash" "*"
```